src/client/react/Components/Admin/AdminEventRow.jsx

```diff
@@ -20,6 +27,41 @@ export default function AdminEventRow(props) {
     hasChanged.current = true;
   }, [event]);
 
+  const accessToken = window.boUserData.access_token
+  const apiUrl = window.boClientConfig.baseUrl
+
+  const getUsers = async () => {
+    if (!loading) {
+      setIsLoading(true);
+
+      await axios.get(`${apiUrl}/event/${event.id}/participants/`, { headers: { authorization: `Bearer ${accessToken}` } })
```

Comment:
Ich würde mir hier wünschen, dass entweder `await` mit `try-catch` oder `.then(...).catch(...)` genutzt wird. Ein Mix aus beiden Varianten macht es unnötig schwerer zu verstehen warum hier Dinge in `.then` passieren und warum das Klicken des CSV-Links unten geschehen.

Reply:
Habe es auf async/await mit try-catch umgebaut.